Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux
+LABEL maintainer="amazon"
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# create directory for the app user
+RUN mkdir -p /home/app
+ENV HOME=/home/app
+ENV APP_HOME=$HOME/workdir
+ENV PYTHONPATH=$APP_HOME/plugins/
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+
+RUN yum update -y
+RUN yum install -y python37 gcc gcc-g++ python3-devel
```

Comment:
Can it be Python 3.9?

Reply:
theoretically yes. The reason for 3.7 is that this is only supported version for MWAA (at least it was when I started) so I wanted it can be used in other projects related to MWAA.